Makefile

```
@@ -167,6 +167,9 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 bundle-build: ## Build the bundle image.
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
+openshift-bundle-build: bundle ## Build the bundle image for OpenShift.
+	@printf "%s\n" '' 'LABEL com.redhat.openshift.versions="v4.6"' 'LABEL com.redhat.delivery.operator.bundle=true' 'LABEL com.redhat.delivery.backport=true' >> bundle.Dockerfile
```

We probably want to increase the openshift versions here? I think without 4.7 and 4.8, they won't appear in the latest catalogs
```suggestion
	@printf "%s\n" '' 'LABEL com.redhat.openshift.versions="v4.6,v4.7,v4.8"' 'LABEL com.redhat.delivery.operator.bundle=true' 'LABEL com.redhat.delivery.backport=true' >> bundle.Dockerfile
```

`A single version preceded by '=' indicates that the operator is supported ONLY on that version of OpenShift. For example, using "=v4.6" will add the operator to the certified catalog for OpenShift 4.6, but not for later OpenShift releases.`

---

There's not a `=` 😅 it's just `4.6` that means 4.6 and after.

---

Never mind me, misread the docs